base/all/tests/Rcheck_reference.log

```diff
@@ -1,29 +1,74 @@
-* using log directory ‘/tmp/Rtmp3UHCCJ/PEcAn.all.Rcheck’
-* using R version 3.5.2 (2018-12-20)
+* using log directory ‘/home/tanishq010/pecan/base/PEcAn.all.Rcheck’
```

Comment:
@moki1202 @nanu1605 `check_with_errors` explicitly ignores these lines, because they're always machine-specific and often session-specific too (note the old version was run in a tempdir). So it's OK to leave these as your machine generates them.